config/jobconfig.toml

```diff
@@ -263,6 +263,13 @@
 "# Spawn Positions" = 5
 "# Total Positions" = 5
 
+[VETERAN_ADVISOR]
+"# Playtime Requirements" = 6000
```

Comment:
I'm like 90% sure this does nothing on a station trait job.

Reply:
This is resolved right, as this is in the trait job code. This shows the trait job sign up button only if person has fulfilled the time requirement.
```c
/datum/station_trait/job/can_display_lobby_button(client/player)
    var/datum/job/our_job = SSjob.GetJob(job_to_add::title)
    return our_job.player_old_enough(player) && ..()`